File: LinearSearch/LinearSearch.java
package com.company;

public class LinearSearch {
    public static int searchPos(int value,int[] array){
        for (int i = 0; i <array.length; i++) {
            if(array[i]==value)
                return i;
        }
        return -1;
    }

    public static boolean hasValue(int value,int[] array) {
        return searchPos(value,array) >= 0;
    }

}

File: LinearSearch/LinearSearch++/LinearSearch.cpp
#include "LinearSearch.h"
#include<iostream>



LinearSearch::LinearSearch()
{
}

int LinearSearch::searchPos(int *array, int value,int size)
{
	for (int i = 0; i <size; i++)
	{
		if (array[i] == value)
			return i;
	}
	return -1;
}

bool LinearSearch::hasValue(int *array, int value, int size)
{
	if(searchPos(array, value,size) >= 0)
		return true;
	return false;
}


LinearSearch::~LinearSearch()
{
}
